pine-a64-plus/armbian.yaml

```diff
@@ -0,0 +1,93 @@
+
+{{- $image := or .image "debos-debian-bullseye-pine-a64.img" -}}
+
+architecture: "arm64"
+
+
+actions:
+  - action: debootstrap
+    suite: "bullseye"
+    components:
+      - main
+      - contrib
+      - non-free
+    mirror: https://deb.debian.org/debian
+    variant: minbase
+
+  - action: apt
+    description: Install some packages
+    packages: 
+        - sudo
+        - openssh-server
+        - isc-dhcp-client
+        - iproute2
+        - ifupdown
+
+# Armbian-specific (kernel, bootloader and settings)
```

Comment:
why not debian ? you can use the `u-boot-sunxi` package for uboot which should have pine64 support and `linux-image-arm64`

Reply:
Just because I have already Armbian instance and just tried to reproduce it using debos for the very first time.

Make it pure upstream Debian is a good idea I think, I will do on next version.